.github/workflows/screener-build.yml

```diff
@@ -97,31 +102,56 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
+      - run: |
+          echo "Is PR build? ${{startsWith(github.ref, 'refs/pull/')}}"
+          packageAffected=$(yarn --silent check:affected-package --packages @fluentui/docs --pr=${{startsWith(github.ref, 'refs/pull/')}})
+          if [[ $packageAffected == false ]]; then
+            echo "Should skip screener"
+            echo "IS_ARTIFACT_PRESENT=false" >> $GITHUB_ENV
+          else
+            echo "IS_ARTIFACT_PRESENT=true" >> $GITHUB_ENV
```

Comment:
Can we stick to `BUILD_SCREENER_ARTIFACT` in this workflow (`screener-build.yml`)? It has a lot of sense in `screener-run.yml`, but not there...